main/zoe/api/zoe-helpers.md

```diff
@@ -342,3 +342,51 @@ import {
 } from '@agoric/zoe/src/contractSupport';
 await saveAllIssuers(zcf, { G: gIssuer, D: dIssuer, P: pIssuer });
 ```
+
+## offerTo(zcf, invitation, keywordMapping, proposal, fromSeat, toSeat)
+- `zcf` `{ContractFacet}`
+- `invitation` `{ERef<Invitation>}`
+- `keywordMapping` `{KeywordKeywordRecord=}`
+- `proposal` `{Proposal}`
+- `fromSeat` `{ZCFSeat}`
+- `toSeat` `{ZCFSeat}`
+- Returns: `{OfferToReturns}`
+
+`offerTo()` makes an offer from your current contract instance (which we'll call
+"contractA") to another contract instance **tyg todo: Do these have to be instances of the
```

Comment:
Good question. Nope, they do not have to be instances of the same contract. Any instance to any other instance.

Reply:
I removed this in a commit and clarified the answer